src/hal/src/pso/mod.rs

```diff
@@ -100,6 +104,10 @@ bitflags!(
         /// Read/Write access from host.
         /// (Not a real pipeline stage)
         const HOST = 0x4000;
+        /// Task shader stage.
+        const TASK = 0x80000;
```

Comment:
should be `TASK_SHADER` and `MESH_SHADER` for consistency with other flags

Reply:
Done.